tests/acceptance/repo-settings-test.js

```diff
@@ -111,13 +111,11 @@ test('view settings', function(assert) {
     assert.equal(settingsPage.environmentVariables(1).value, '••••••••••••••••');
 
     assert.equal(settingsPage.crons(0).branchName, 'daily-branch');
-    assert.equal(settingsPage.crons(0).interval, 'daily');
-    assert.ok(settingsPage.crons(0).nextEnqueuing.indexOf(':19'), 'expected cron next enqueuing to match');
+    assert.ok(settingsPage.crons(0).nextEnqueuing.indexOf('Every') === 0, 'expected cron next enqueuing to match');
     assert.ok(settingsPage.crons(0).disableByBuildText.indexOf('Even') === 0, 'expected cron to run even if no new commit after last build');
 
     assert.equal(settingsPage.crons(1).branchName, 'weekly-branch');
-    assert.equal(settingsPage.crons(1).interval, 'weekly');
-    assert.ok(settingsPage.crons(1).nextEnqueuing.indexOf(':11'), 'expected cron next enqueuing to match');
+    assert.ok(settingsPage.crons(1).nextEnqueuing.indexOf('Every') === 0, 'expected cron next enqueuing to match');
```

Comment:
Can we distinguish between the assertion messages in these two cases?  Maybe something like "Shows weekly/daily enqueuing text" or something to that effect?